cmd/configure/configure.go

```diff
@@ -68,7 +68,7 @@ func execute(cmd *cobra.Command) {
 }
 
 func createConfigFileIfNotExist() {
-	dirPath := path.Join(os.Getenv("HOME"), "." + app.App.Name)
+	dirPath := path.Join(os.Getenv("HOME"), "."+app.App.Name)
```

Comment:
why this change? the previous formatting looks better

Reply:
yes. the previous one is better. but running `golangci-lint run -E gofmt -E gci --fix` did this.